libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -24,14 +27,34 @@ import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
       :host {
         display: block;
       }
+
+      .grey-color {
+        color: var(--watt-color-neutral-grey-900);
+      }
     `,
   ],
-  template: `<p>You do not have any metering points.</p>`,
+  template: `<ng-container *rxLet="meteringPoints$ as meteringPoints">
```

Comment:
suggestion: I would pull this out into its own "eo-metering-point-list" component, and clean up the shell + prepare for more logic to come on this page.

Reply:
Done :)